packages/config/package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "@cgr/config",
```

Comment:
not sure about the naming. `config` is too wide. Do we have other config than Eslint?